### PR TITLE
Release latest generation changes

### PR DESCRIPTION
**Pull request by Ndiritu**
Releasing latest PHP generation changes along with other dependency version bumps.

- Bump submodules/vipr from ff47cf1 to c5c11d1
- Bump NLog from 4.7.9 to 4.7.10 
- Bump submodules/vipr from c5c11d1 to 48eb08c
- Bump NLog.Config from 4.7.9 to 4.7.10 
- Fix type issues in docblocks
- Bump Microsoft.NET.Test.Sdk from 16.9.4 to 16.10.0
- Bump MSTest.TestFramework from 2.2.3 to 2.2.4 
- Bump MSTest.TestAdapter from 2.2.3 to 2.2.4
- Bump submodules/vipr from 48eb08c to 1b0afae 

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/pull/540)

### CODE REVIEW

**Review by SilasKenneth — APPROVED** [review]

**Comment by Ndiritu** [comment]
Need an approval from a code-owner to merge @baywet @zengin @MIchaelMainer

**Review by zengin — APPROVED** [review]

**Pull request merged**
